### ISSUE TITLE
Zotero Manually Add Citation to Bibliography

### ISSUE THREAD

**Issue by 17hoehbr** [ONLYOFFICE/onlyoffice.github.io#359]
Hello, I have been frequently using the Zotero plugin lately to write APA papers. The in text citation editor works fine and properly adds the citation to the bibliography, but with APA style it sometimes makes more sense to write the in-text citation myself for short quoting.

Example:
``` According to Jones (1998), "students often had difficulty using APA style, especially when it was their first time" (p. 199). ```
``` Jones (1998) found "students often had difficulty using APA style" (p. 199); what implications does this have for teachers? ```

In these situations, I type the citation myself instead of using the insert citation tool, but then it does not get added to the bibliography. I don't think it would be feasible to somehow make the plugin aware of these short citations, but it would be nice if there was at least an option to manually add sources to the bibliography section that do not link to in text citations created by Zotero. If this is already possible please let me know because I have not found an option to do so.

**Comment by VladimirProkofiev52** [comment]
Hello! @17hoehbr 
Just to confirm, are you looking to input text into a document and then add it to the Zotero site via a plugin?

**Comment by 17hoehbr** [comment]
No, I am only looking to add bibliography information from existing Zotero entries to the bibliography section. I would just like an option to manually add entries that are not associated with an in text citation.

Example:
![image](https://github.com/user-attachments/assets/86ab8c9f-91f6-4b74-be6e-5a370176a486)

Zotero does not recognize Hassenzahl (2006) as a citation since I typed in manually. Thus, clicking insert citation just gives the message "Please insert some citation into the document". I would like to have some way to manually add a source to the bibliography even if it is not associated with an in text citation generated by the Zotero plugin.
